src/app/authentication/login.component.css

```diff
@@ -0,0 +1,35 @@
+.header-1 {
+	color : #04529D;
```

Comment:
probably easier to use scss/less so that you can create functions/variables in css, easier for coding & readability